samples/ClientCredentialsFlow/ClientApp/Program.cs

```diff
@@ -15,14 +16,26 @@ public class Program
         {
             var client = new HttpClient();
 
-            var token = await GetTokenAsync(client);
-            Console.WriteLine("Access token: {0}", token);
-            Console.WriteLine();
-
-            var resource = await GetResourceAsync(client, token);
-            Console.WriteLine("API response: {0}", resource);
+            try
+            {
+                var token = await GetTokenAsync(client);
+                Console.WriteLine("Access token: {0}", token);
+                Console.WriteLine();
 
-            Console.ReadLine();
+                var resource = await GetResourceAsync(client, token);
+                Console.WriteLine("API response: {0}", resource);
+                Console.ReadLine();
+            }
+            catch (System.Net.Http.HttpRequestException ex)
+            {
+                var builder = new StringBuilder();
+                builder.AppendLine("+++++++++++++++++++++");
+                builder.AppendLine(ex.Message);
+                builder.AppendLine(ex.InnerException.Message);
+                builder.AppendLine("Make sure you started the authorization server.");
+                builder.AppendLine("+++++++++++++++++++++");
+                Console.WriteLine(builder.ToString());
+            }
```

Comment:
If you like this, let me know, and I will add it also to the password flow sample.